### ISSUE TITLE
Missing first timestamp?

### ISSUE THREAD

**Issue by morganherlocker** [sharedstreets/trip-simulator#27]
I received a bug report that occasionally the first timestamp is not populated or possibly null.

**Comment by gautamarora** [comment]
Possibly related to this line: https://github.com/sharedstreets/trip-simulator/blob/master/src/cli.js#L42

currently:
```
const start = +argv.seconds;
```

should be:

```
const start = +argv.start;
```

**Comment by morganherlocker** [comment]
Good catch, this could cause all sorts of issues - looks like a typo I made during the last refactor.
